wpimath/src/main/java/edu/wpi/first/math/estimator/DifferentialDrivePoseEstimator.java

```diff
@@ -218,11 +226,30 @@ private Matrix<N5, N1> f(Matrix<N5, N1> x, Matrix<N3, N1> u) {
    * @param gyroAngle The angle reported by the gyroscope.
    */
   public void resetPosition(Pose2d poseMeters, Rotation2d gyroAngle) {
+    resetPosition(poseMeters, gyroAngle, 0, 0);
+  }
+
+  /**
+   * Resets the robot's position on the field.
+   *
+   * <p>The gyroscope angle does not need to be reset here on the user's robot code. The library
+   * automatically takes care of offsetting the gyro angle.
+   *
+   * @param poseMeters The position on the field that your robot is at.
+   * @param gyroAngle The angle reported by the gyroscope.
+   * @param leftDistanceMeters The distance traveled by the left encoder.
+   * @param rightDistanceMeters The distance traveled by the right encoder.
```

Comment:
```suggestion
   * @param leftPositionMeters The distance traveled by the left encoder.
   * @param rightPositionMeters The distance traveled by the right encoder.
```

Reply:
This has been replaced with the correct param names.